Dockerfile

```
@@ -5,6 +5,7 @@
 FROM ubuntu:focal AS python-dependencies
 RUN apt-get update && apt-get install --no-install-recommends --yes python3-pip python3-setuptools
 COPY requirements.txt /tmp/requirements.txt
+RUN pip3 config set global.disable-pip-version-check true
 RUN --mount=type=cache,target=/root/.cache/pip pip3 install --user --requirement /tmp/requirements.txt
```

```suggestion
RUN --mount=type=cache,target=/root/.cache/pip env DISABLE_PIP_VERSION_CHECK=1 pip3 install --user --requirement /tmp/requirements.txt
```

might be even quicker (doesn't actually run pip3 config; therefore avoiding a Python interpreter)

---

Lol, I created GitBulk with the help of a keyboard ⌨️ and my brain 🧠  as well 😉 .

I was going to put it in one line with `pip3 --disable-pip-version-check` but then I thought that we might want to use pip multiple times in the Dockerfile so it might be better to just disable it globally, the other option is to set the env variable globally as well